mockgcp/apis/mockgcp/cloud/ids/v1/service.proto

```diff
@@ -13,187 +13,285 @@
+  // Required. Lowest threat severity that this endpoint will alert on.
   optional string severity = 10 [json_name="severity"];
+  // Output only. Current state of the endpoint.
   optional string state = 11 [json_name="state"];
+  // List of threat IDs to be excepted from generating alerts.
   repeated string threat_exceptions = 12 [json_name="threatExceptions"];
+  // Whether the endpoint should report traffic logs in addition to threat logs.
   optional bool traffic_logs = 13 [json_name="trafficLogs"];
+  // Output only. The update time timestamp.
   optional .google.protobuf.Timestamp update_time = 14 [json_name="updateTime"];
 }
+
+  // Represents a textual expression in the Common Expression Language (CEL) syntax. CEL is a C-like expression language. The syntax and semantics of CEL are documented at https://github.com/google/cel-spec. Example (Comparison): title: "Summary size limit" description: "Determines if a summary is less than 100 chars" expression: "document.summary.size() < 100" Example (Equality): title: "Requestor is owner" description: "Determines if requestor is the document owner" expression: "document.owner == request.auth.claims.email" Example (Logic): title: "Public documents" description: "Determine whether the document should be publicly visible" expression: "document.type != 'private' && document.type != 'internal'" Example (Data Manipulation): title: "Notification string" description: "Create a notification string with a timestamp." expression: "'New message received at ' + string(document.create_time)" The exact variables and functions that may be referenced within an expression are determined by the service that evaluates it. See the service documentation for additional information.
 message Expr {
+  // Optional. Description of the expression. This is a longer text which describes the expression, e.g. when hovered over it in a UI.
   optional string description = 1 [json_name="description"];
+  // Textual representation of an expression in Common Expression Language syntax.
   optional string expression = 2 [json_name="expression"];
+  // Optional. String indicating the location of the expression for error reporting, e.g. a file name and a position in the file.
   optional string location = 3 [json_name="location"];
+  // Optional. Title for the expression, i.e. a short string describing its purpose. This can be used e.g. in UIs which allow to enter the expression.
   optional string title = 4 [json_name="title"];
 }
+
 message ListEndpointsResponse {
+  // The list of endpoints response.
   repeated Endpoint endpoints = 1 [json_name="endpoints"];
+  // A token, which can be sent as `page_token` to retrieve the next page. If this field is omitted, there are no subsequent pages.
   optional string next_page_token = 2 [json_name="nextPageToken"];
+  // Locations that could not be reached.
   repeated string unreachable = 3 [json_name="unreachable"];
 }
+
+  // The response message for Locations.ListLocations.
```

Comment:
Can we fix the indentation for top tier message?

Reply:
Good catch, fixed